### PR TITLE
Update oc get version

### PR DESCRIPTION
**Pull request by ebattat**
Fix:
1. Update oc get version using  -o jsonpath instead of jq parser due to change in json hierarchy

### CODE REVIEW

**Comment by ebattat** [comment]
Due to mix branch

File: benchmark_runner/jupyterlab/templates/run_workload/how_to_run_workload.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# To run a workload with custom parameters through JupyterLab terminal, follow these steps:
```
Comment (RobertKrawitz) [inline benchmark_runner/jupyterlab/templates/run_workload/how_to_run_workload.md:1]: This file does not belong in this PR.

**Review by RobertKrawitz — COMMENTED** [review]
Need to clean up the PR; it looks like it has your previous PR also incorporated.
